src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -718,8 +745,8 @@ class ActivityResourceIntTest : IntegrationTestBase() {
               courseScheduleRule()
             }
           }
-          offender(nomsId = "A1234TT") {
```

Comment:
Use the default nomsId wherever possible, it's largely irrelevant to what is being tested and is just noise.